deliverables/ruby_monk_problems/calculator.rb

```diff
@@ -1,9 +1,9 @@
 class Calculator
   def add(a, b)
-   # your code here
+ a+b
```

Comment:
You usually want to tab your code so that it is more readable. See https://github.com/bbatsov/ruby-style-guide for more info.